README.md

```diff
@@ -33,5 +39,33 @@ Open up [http://localhost:3000](http://localhost:3000) and you should be ready t
 
 If you're used to using the `vercel dev` command provided by [Vercel CLI](https://vercel.com/cli) instead, you can also use that, but it's not needed.
 
-
 <img align="center" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSagMR3FC2rPHvoqZFjHunC5DkmFLdeNzok9d7P8OHW&s" width="50%" alt="Vercel" >
+
+## Contributing - [look at our doc here](./CONTRIBUTING.md)
+
+## Formatting and Linting
+
+Please utilize our formatting and linting command `npm run lint`. Also included in our documentation, are great instructions on how to [setup this functionality to automatically run on save in VS code](./docs/formatting-and-linting.md).
+
+## Prisma
+
+We've created some handy scripts to help with database management with Prisma. Occasionally you might need to use these to update the database on your local machine.
+
+- `npm run build:db` - [generate your prisma client](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client) when starting out or when updates are made to the Prisma schema
+
+- `npm run update:db` - [prototype your schema](https://www.prisma.io/docs/concepts/components/prisma-migrate/db-push) to iterate on schema design locally
+
+- `npm run seed:db` - [consistently create data by seeding](https://www.prisma.io/docs/guides/migrate/seed-database) data into our database. We have a slightly different setup than when is in the Prisma docs. We are using `--require tsconfig-paths/register` to use the `~` path feature in Remix ([ref: Kent C. Dodds](https://github.com/remix-run/blues-stack/issues/143#issuecomment-1515339235))
+
+## Deployment
```

Comment:
I think this section can be removed. It describes how to deploy a new Remix project.

Reply:
Thanks for sharing this explanation from Kent. That's good to know.